packages/paste-core/components/toast/src/types.ts

```diff
@@ -0,0 +1,42 @@
+import * as PropTypes from 'prop-types';
+import {ValueOf} from '@twilio-paste/types';
+
+export const ToastIconVariants = {
+  ERROR: 'error',
+  NEUTRAL: 'neutral',
+  SUCCESS: 'success',
+  WARNING: 'warning',
+} as const;
+
+export const ToastRoles = {
```

Comment:
For toasts lets make them all role="status". For alerts it's fine as the idea is that they are not stacked. For toasts it's different, as they are designed to be stacked. A role of alert is an assertive live region, meaning it will often jump the queue when being announced by a screen reader. If you have 2 toasts and the last one is assertive, it'll cause the first one not to be announced.

Reply:
Got it. Thanks for the explanation.